frontend/public/components/broker-management.tsx

```diff
@@ -1,19 +1,13 @@
 import * as React from 'react';
```

Comment:
This entire file should be removed as well. Service catalog includes

Service Brokers
Service Classes
Service Instances
Service Bindings

Be careful about service bindings since there is another kind of service binding that is used in the dev console that should *not* be removed. Only the service catalog bindings should be removed.

We also need to remove the protractor tests for service catalog and any models as well as the custom table filters, YAML templates, and feature flags.